card/lib/card/mod/module_template.rb

```diff
@@ -33,33 +41,45 @@ def simple_load?
 
       private
 
+      # find all comment lines at the beginning of a mod file, up to the first
+      # non-comment line.  (These will be inserted before the module declaration,
+      # so that Yard will interpret them as a module comment.)
+      def capture_module_comment
+        content_lines = @content.split "\n"
+        comment_lines = []
+
+        content_lines.each do |line|
+          comment?(line) ? comment_lines << content_lines.shift : break
+        end
+
+        @content = content_lines.join "\n"
+        @module_comment = comment_lines.join "\n"
+      end
+
+      def comment? line
+        line.match?(/^ *\#/)
```

Comment:
Have you considered `#! no set module` comments in the first line?  (idk if they can appear in this place or if it's a problem to move them)

Reply:
Or utf-8 comments?

Reply:
I noted this in slack, but I'm confused about `no set module`. It kinda looks like it isn't actually doing anything right now.

As to your question: I'm not entirely sure, but I know:
1. none of the comment stuff takes any effect unless you're doing tmp files. 
2. usually we add the utf8 stuff automatically after all the processing.
3. my _guess_ is that yardoc ignores `#!` 

So, it's possible that something non-ideal happens in tmp files, but maybe not, and in any case it's an edge case.  My inclination is not to worry about it until we see something we don't like in a doc.